tox.ini

```diff
@@ -6,25 +6,20 @@ ignore_basepython_conflict = true
 # execute `tox` in the command-line
 # bellow you will find explanations for all environments
 envlist =
-    py37
-    py38
-    py39
+    test
```

Comment:
again this is not optimal. this means locally you will have to test everything together instead of having control for each thing you want to test. likely you will have "interpreter not found errors". You can setup this in the gitactions. there's a way to set `tox -e py**` in coordination with the matrix in the github actions. Look for it. :wink:

Reply:
True, but as I tried to say in my other comment usually you have only one version and if yes you can decide by hand which version you would like to test. Usually you want to develop for the python version you are working with.

Reply:
I will end up liking it at the end :wink:

Reply:
> a way to set tox -e py** in coordination with the matrix in the github actions

Try the below:

```
pyver="py$(echo '${{ matrix.python-version }}' | tr -d . )"
tox -e $pyver
```

Example in https://github.com/lilyminium/mdacli/blob/fc221f3e85558d70c46d6223788196d86d41e7b2/.github/workflows/ci.yml#L31-L34